src/mygrad/nnet/activations/softmax.py

```diff
@@ -9,48 +9,48 @@ def _softmax(x, kwargs):
     x = x - x.max(**kwargs)
     if np.issubdtype(x.dtype, np.integer):
         x = x.astype(np.float)
-    np.exp(x, out=x)
-    x /= x.sum(**kwargs)
+    if x.ndim > 0:
+        np.exp(x, out=x)
+        x /= x.sum(**kwargs)
+    else:
+        x = np.exp(x)
+        x = x / x.sum(**kwargs)
```

Comment:
isn't this branch just for scalars? In which case the output of softmax would just be 1 by definition so we don't need to do this computation?

Reply:
oh... ha. You are right